libs/api/Verifier.h

```diff
@@ -17,6 +17,7 @@ load_byte_code(
     _In_opt_z_ const char* sectionname,
     _In_ ebpf_verifier_options_t* verifier_options,
     _Out_ std::vector<ebpf_program_t*>& programs,
+    _Out_opt_ std::vector<ebpf_map_t*>& maps,
```

Comment:
I don't think `&` works with nullptr, so `_opt_` would not be needed.  Have you found otherwise?

Reply:
Changed this to _Inout_